packages/terra-form-checkbox/src/CheckboxField.jsx

```diff
@@ -113,7 +113,15 @@ const CheckboxField = (props, { intl }) => {
     <legend id={legendAriaDescriptionId} className={cx(['legend-group', { 'legend-group-hidden': isLegendHidden }])}>
       <div {...legendAttrs} className={legendClassNames}>
         {isInvalid && <span className={cx('error-icon')} />}
-        {required && (isInvalid || !hideRequired) && <span className={cx('required')}>*</span>}
+        {required && (isInvalid || !hideRequired) && (
+          <React.Fragment>
+            <div aria-hidden="true" className={cx('required')}>*</div>
+            <span className={cx('visually-hidden-text')}>
+              {intl.formatMessage({ id: 'Terra.form.field.required' })}
+              {' '}
```

Comment:
what's the string for?

Reply:
It adds a space between the word required and the fieldset legend.
<img width="1438" alt="Screen Shot 2019-05-22 at 3 15 58 PM" src="https://user-images.githubusercontent.com/633148/58205659-99e65280-7ca4-11e9-96ec-257340b1bc6d.png">